packages/client/src/scripts/postinstall.ts

```diff
@@ -0,0 +1,393 @@
+import Debug from '@prisma/debug'
```

Comment:
In there we are now taking advantage of TypeScript, and we compile the postinstall script with esbuild. The diff show better in the original commit [`ef49571` (#13702)](https://github.com/prisma/prisma/pull/13702/commits/ef49571235807986558560e025b401813554bc64)